src/store/misions/missions.js

```diff
@@ -1,9 +1,62 @@
+const BASE_URL = 'https://api.spacexdata.com/v3/missions';
 const LOAD_MISSIONS = 'missions/load';
+const JOIN_MISSION = 'sissions/join';
+const LEAVE_MISSION = 'sissions/leave';
+
+const displayMissions = (payload) => ({
+  type: LOAD_MISSIONS,
+  payload,
+});
+
+export const joinMission = (id) => ({
+  type: JOIN_MISSION,
+  id,
+});
+
+export const leaveMission = (id) => ({
+  type: LEAVE_MISSION,
+  id,
+});
+
+export const fetchMissions = async (dispatch) => {
+  const res = await fetch(BASE_URL);
+  const missions = await res.json();
+  console.log(missions);
```

Comment:
- Please note that console logs can be helpful during development, but leaving them in your code when you deploy your application can negatively affect performance, security, maintenance, and clarity. 
- **[OPTIONAL]** It is generally recommended to remove all console logs from your production code.